### PR TITLE
Documentation: Update example in writing-custom-queries.md to include…

### PR DESCRIPTION
**Pull request by timflannagan1**
… timestamps.

### CODE REVIEW

**Comment by timflannagan1** [comment]
might be worthwhile explaining why `spec.inputs` is necessary for the `period_end`, `period_start` columns in the schema definition.

File: Documentation/writing-custom-queries.md [hunk 1]
```diff
@@ -212,19 +212,30 @@ metadata:
...
+  - name: avg_replica_unready_seconds
     type: double
+  inputs:
+  - name: ReportingStart
+  - name: ReportingEnd
   query: |
     SELECT
-        labels['namespace'] as namespace,
-        labels['deployment'] as deployment,
+        timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart | prestoTimestamp |}' AS period_start,
```
Comment (chancez) [inline Documentation/writing-custom-queries.md]: I was thinking we would add these in the next example. This one is intended to work without input.

Comment (chancez) [reply Documentation/writing-custom-queries.md]: Add adding an explanation to the section below:

> However, the above example is missing one crucial bit, and that's the ability to constrain the period of time that this query is reporting over.

Updating that to also mention that we need to expose the period being reported over (in addition to filtering)

Comment (timflannagan1) [reply Documentation/writing-custom-queries.md]: I moved the timestamp inputs out of the non-input example and added them to the final `ReportGenerationQuery` snippet later in the doc. 

I added a link to the [reports.md](https://github.com/operator-framework/operator-metering/blob/master/Documentation/report.md#reportingstart) that covers ReportingStart, ReportingEnd, and spec.inputs.

**Comment by chancez** [comment]
@timflannagan1 This looks great. Can you squash your commits then I'll approve it.

**Comment by timflannagan1** [comment]
@chancez done

**Pull request merged**